data/net/minecraft/world/inventory/AbstractContainerMenu.mapping

```diff
@@ -39,13 +39,13 @@ CLASS net/minecraft/world/inventory/AbstractContainerMenu
 		ARG 1 player
 		ARG 2 id
 	METHOD clicked (IILnet/minecraft/world/inventory/ClickType;Lnet/minecraft/world/entity/player/Player;)V
-		ARG 1 mouseX
-		ARG 2 mouseY
+		ARG 1 slotId
+		ARG 2 dragType
```

Comment:
The crash report detail in the method seems to imply that this (and the same parameter in `doClick`) is for the mouse button, not the 'drag type' (which is also unclear as a name, in what it refers to).